ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -550,9 +560,25 @@ namespace ix
                 std::string reason(_rxbuf.begin()+ws.header_size + 2,
                                    _rxbuf.begin()+ws.header_size + (size_t) ws.N);
                 
-                bool remote = true;
+                
+                // We receive a CLOSE frame from remote and are NOT the ones who triggered the close
```

Comment:
I'm skeptical on our ability to know that the close is remote, by just checking _readyState. 

Can CLOSING state be only set when we close ourself ?

Reply:
All the logic is based on that, the CLOSING state is only set in the close() method so when we close the connection by ourself